<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(164, 161, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>601</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#f8fc06;&quot;&gt;Q-FADD Pre-Processing&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="FileIOframe">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>571</width>
          <height>451</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="1,0,0,0,0">
         <item row="7" column="1">
          <widget class="QLineEdit" name="ProtChannelLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="MovieLabel">
           <property name="text">
            <string>Movie File:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QLineEdit" name="MovieFileLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QRadioButton" name="BatchModeButton">
           <property name="text">
            <string>Batch Mode</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QLineEdit" name="OutDirectLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2" colspan="2">
          <widget class="QRadioButton" name="SingleFileButton">
           <property name="text">
            <string>Single File</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="8" column="2" colspan="2">
          <widget class="QLineEdit" name="ROIXBufferLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="ProtChannelLabel">
           <property name="text">
            <string>Accumulation Channel:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="NucChannelLabel">
           <property name="text">
            <string>Nucleus Tracking Channel:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="4">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>pixels</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="QLineEdit" name="ROIFileLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QPushButton" name="BatchDirectoryButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="3">
          <widget class="QLineEdit" name="BatchDirectoryLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="4">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>pixels</string>
           </property>
          </widget>
         </item>
         <item row="10" column="4">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>frames</string>
           </property>
          </widget>
         </item>
         <item row="10" column="2" colspan="2">
          <widget class="QLineEdit" name="IrradFrameLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Pre-Irradiation Frames:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>File Directory:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="5">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline;&quot;&gt;File Input/Output&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="13" column="2">
          <widget class="QCheckBox" name="BleachCheckBox">
           <property name="text">
            <string>Correct for Bleaching</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2" colspan="2">
          <widget class="QLineEdit" name="ROIYBufferLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="OutDirectLabel">
           <property name="text">
            <string>Output Directory:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>ROI Y Buffer:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Additional ROI's:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="13" column="1">
          <widget class="QCheckBox" name="DriftCheckBox">
           <property name="text">
            <string>Correct for Drift</string>
           </property>
          </widget>
         </item>
         <item row="12" column="2" colspan="2" alignment="Qt::AlignLeft">
          <widget class="QSpinBox" name="AdditionalRoiSpinbox">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="maximum">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="NucChannelLineEdit">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>ROI X Buffer:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QPushButton" name="ROIBrowserButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Running Format:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="textFormat">
            <enum>Qt::RichText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QPushButton" name="MovieBrowserButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="ROILabel">
           <property name="text">
            <string>ROI File:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="5" column="4">
          <widget class="QPushButton" name="OutputDirectoryButton">
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
         <item row="14" column="3" colspan="2">
          <widget class="QPushButton" name="SaveButton">
           <property name="text">
            <string>Save Options</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SubmitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>BatchModeButton</tabstop>
  <tabstop>SingleFileButton</tabstop>
  <tabstop>BatchDirectoryLineEdit</tabstop>
  <tabstop>BatchDirectoryButton</tabstop>
  <tabstop>MovieFileLineEdit</tabstop>
  <tabstop>MovieBrowserButton</tabstop>
  <tabstop>ROIFileLineEdit</tabstop>
  <tabstop>ROIBrowserButton</tabstop>
  <tabstop>OutDirectLineEdit</tabstop>
  <tabstop>NucChannelLineEdit</tabstop>
  <tabstop>ProtChannelLineEdit</tabstop>
  <tabstop>ROIXBufferLineEdit</tabstop>
  <tabstop>ROIYBufferLineEdit</tabstop>
  <tabstop>DriftCheckBox</tabstop>
  <tabstop>BleachCheckBox</tabstop>
  <tabstop>SubmitButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
